webapp/src/components/App/Singlestudy/explore/Modelization/Areas/Hydro/Allocation/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box } from "@mui/material";
+import { useOutletContext } from "react-router";
+import { t } from "i18next";
+import { AxiosError } from "axios";
+import Form from "../../../../../../../common/Form";
+import Fields from "./Fields";
+import { StudyMetadata } from "../../../../../../../../common/types";
+import useAppSelector from "../../../../../../../../redux/hooks/useAppSelector";
+import { getCurrentAreaId } from "../../../../../../../../redux/selectors";
+import {
+  AllocationFormFields,
+  getAllocationFormFields,
+  setAllocationFormFields,
+} from "./utils";
+import { SubmitHandlerPlus } from "../../../../../../../common/Form/types";
+import useEnqueueErrorSnackbar from "../../../../../../../../hooks/useEnqueueErrorSnackbar";
+
+function Allocation() {
+  const {
+    study: { id: studyId },
+  } = useOutletContext<{ study: StudyMetadata }>();
+  const areaId = useAppSelector(getCurrentAreaId);
+  const enqueueErrorSnackbar = useEnqueueErrorSnackbar();
+
+  ////////////////////////////////////////////////////////////////
+  // Event handlers
+  ////////////////////////////////////////////////////////////////
+
+  const handleSubmit = (data: SubmitHandlerPlus<AllocationFormFields>) => {
+    try {
```

Comment:
`return setAllocationFormFields(...);`, then Form component will show the error.

You don't know the cause of the error, it's not necessarily because of the deletion

Reply:
the Form can handle errors without a catch ?